src/containers/shared/components/Currency.tsx

```diff
@@ -8,16 +8,20 @@ export interface Props {
   issuer?: string
   currency: string
   link?: boolean
+  showIssuer?: boolean
 }
 
 const Currency = (props: Props) => {
-  const { issuer, currency, link = true } = props
+  const { issuer, currency, link = true, showIssuer = true } = props
```

Comment:
rename this to something like shorten or abbreviate.  Since we are still showing part of the issuer.

Reply:
`truncate` perhaps?

Reply:
done